### PR TITLE
Blog Template

### PR DESCRIPTION
**Pull request by empeje**
Blog Template, try with

```bash
bundle install
bundle exec jekyll serve
```

then open

`localhost:4000`

example

screen shot 2018-05-09 at 2 24 26 am
screen shot 2018-05-09 at 2 24 27 am
screen shot 2018-05-09 at 2 24 28 am
screen shot 2018-05-09 at 2 24 30 am
screen shot 2018-05-09 at 2 24 33 am

### CODE REVIEW

File: _includes/contribute_button.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<section id="downloads" style="margin-bottom: 0px">
```
Comment (xyb994) [inline _includes/contribute_button.html]: Looks like forgot to change the template's `id="downloads"`

File: _includes/repository_button.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<section id="downloads" style="margin-bottom: 0px">
```
Comment (xyb994) [inline _includes/repository_button.html]: same as above

File: _includes/top_nav.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<section id="downloads" style="margin-bottom: 0px">
```
Comment (xyb994) [inline _includes/top_nav.html]: same as above

File: _includes/top_nav.html [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+    <a href="/" class="btn">Home</a>
+    <a href="/alldocs" class="btn">Docs</a>
+    <a href="/allposts" class="btn">Posts</a>
```
Comment (xyb994) [inline _includes/top_nav.html]: Do we want to call it `Blog` here?

**Review by xyb994 — COMMENTED** [review]
Looks like a good start :)

File: _includes/contribute_button.html (hunk 1 above)
Comment (empeje) [reply _includes/contribute_button.html]: thanks, will change it

File: _includes/repository_button.html (hunk 2 above)
Comment (empeje) [reply _includes/repository_button.html]: thanks, will change it

File: _includes/top_nav.html (hunk 3 above)
Comment (empeje) [reply _includes/top_nav.html]: thanks, will change it

File: _includes/top_nav.html (hunk 4 above)
Comment (empeje) [reply _includes/top_nav.html]: Ah you're right.

**Review by dogi — APPROVED** [review]

**Review by xyb994 — APPROVED** [review]

**Review by sahilph — APPROVED** [review]

**Comment by i5o** [comment]
Is there any way we could remove the second "./"?
image

**Review by i5o — APPROVED** [review]
looks good, please take a look at https://github.com/treehouses/treehouses.github.io/pull/10 once it's merged

**Pull request merged**